app/src/main/res/layout/link_example_activity.xml

```diff
@@ -4,6 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/blue"
```

Comment:
do we want to keep it blue?

Reply:
The reason is to identify the white splashes, if it happens. what do you think?

Reply:
Let's roll back it, bc there is a blue CTA button and gray text. Blue bg is ok for local testing but not for permanent use.